HenHen.Framework/Graphics/Sprite.cs

```diff
@@ -0,0 +1,29 @@
+﻿using HenHen.Framework.Extensions;
+
+namespace HenHen.Framework.Graphics
+{
+    class Sprite : Drawable
+    {
+        public ColorInfo Color { get; set; } = new ColorInfo(255, 255, 255);
+        private Raylib_cs.Texture2D texture;
+
+        protected void LoadTexture(string path)
+        {
+            texture = Raylib_cs.Raylib.LoadTexture(path);
+        }
+        protected override void OnRender()
+        {
+            base.OnRender();
+            Raylib_cs.Raylib.DrawTexture(texture, (int)GetRenderPosition().X, (int)GetRenderPosition().Y, Color.ToRaylibColor());
```

Comment:
Ta funkcja Rayliba nie daje możliwości ustawienia jaką wielkość przyjmie tekstura na ekranie.

Reply:
Należy użyć `DrawTexturePro`
Parametry:
- `sourceRec` - którą część tekstury chcemy wyświetlić. Domyślnie zawsze chcemy wyświetlić całą, więc będzie to prostokąt zaczynający się w pozycji (0, 0) i o wielkości tekstury.
- `destRec` - prostokąt definiujący gdzie na ekranie ma pojawić się tekstura.
- `origin` - punkt, wokół którego tekstura będzie obracana. Na razie nie zamierzamy implementować obracania, więc można ustawić na np. `Vector2.Zero`.
- `rotation` - pozostawić jako `0`

Reply:
przydałoby się nie wywoływać dwukrotnie funkcji typu `GetRenderCosTam`, ponieważ wykonują one za każdym razem obliczenia. Najlepiej przypisać zwracaną przez nie wartość do zmiennej lokalnej (`var rect = GetRenderRect()`)